Source/WebCore/rendering/BackgroundPainter.h

```diff
@@ -34,6 +34,12 @@ enum class ShadowStyle : uint8_t;
 
 class FloatRoundedRect;
 
+enum BaseBackgroundColorUsage {
```

Comment:
Can we use enum class for this one as well? Or you'll have a separate PR ready for that?